backend/src/main/kotlin/app/ehrenamtskarte/backend/verification/webservice/schema/CardMutationService.kt

```diff
@@ -119,7 +120,8 @@ class CardMutationService {
         dfe: DataFetchingEnvironment,
         project: String,
         encodedCardInfos: List<String>,
-        generateStaticCodes: Boolean
+        generateStaticCodes: Boolean,
+        applicationIdToMarkAsProcessed: Int? = null
```

Comment:
(optional) i wonder whether we need to send the `applicationId` to the backend if we don't do anything with it. Wouldn't be a boolean sufficient?

Reply:
Yes we do find the right application with it. Without sending the applicationId we only create the card and do not know which application it belongs to. If we don't do anything with it (e.g. for nuernberg or when creating a card that does not belong to a application) we can just pass null.